bgpd/bgp_evpn_vty.c

```diff
@@ -2692,10 +2692,10 @@ DEFUN (no_bgp_evpn_default_originate,
 	return CMD_SUCCESS;
 }
 
```

Comment:
about the change of workflow, is there some user documentation where that information can be written somewhere ?